iib/web/models.py

```diff
@@ -1049,6 +1071,99 @@ def get_mutable_keys(self):
         return rv
 
 
+class RequestMergeIndexImage(Request, RequestIndexImageMixin):
+    """An "merge-index-image" build request."""
+
+    __tablename__ = 'request_merge_index_image'
+
+    id = db.Column(db.Integer, db.ForeignKey('request.id'), autoincrement=False, primary_key=True)
+    source_from_index_id = db.Column(db.Integer, db.ForeignKey('image.id'), nullable=True)
+    source_from_index = db.relationship('Image', foreign_keys=[source_from_index_id], uselist=False)
+    target_from_index_id = db.Column(db.Integer, db.ForeignKey('image.id'), nullable=True)
```

Comment:
Could you please rename target_from_index to target_index here too?

Reply:
done